README.md

```diff
@@ -1,23 +1,23 @@
-
-
-The latest version is 3.1. Feel free to give it a try:
+The latest version is 3.1. Feel free to give it a try using one of our builds:
 
 * Docker image: https://hub.docker.com/r/ripencc/rpki-validator-3-docker
```

Comment:
Maybe proceed with new docker already? Though this one already has many downloads.